support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/subheading.tsx

```diff
@@ -54,18 +56,25 @@ const getSubHeadingCopy = (
 		const tier3HeadingCopy = (
 			<span>{`You'll receive a confirmation email containing everything you need to know about your subscription, including additional emails on how to make the most of your subscription.${' '}`}</span>
 		);
+		const guardianAdLiteCopy = (
+			<span>{`Your valued support powers our journalism${' '}`}</span>
+		);
+		const notSignedInCopy = isTier3
```

Comment:
In some places, eg here, the logic feels quite extensive. Maybe a mapping from product to message might be better here?

Reply:
Good idea, think this would require a rewrite to pass through product throughout the thankyou pages. Possibly outside the the scope of this PR as its big enough at the moment?  Could create a seperate future TrelloCard to look into this?

Reply:
Ok, have added a `thankyouMessage` to the `productCatalogDescription` so it can be looked up via the `productKey`.